docs/src/utils/developers.json

```diff
@@ -0,0 +1,34 @@
+{
+    "bensku": {
+        "avatar": "https://avatars0.githubusercontent.com/u/4330456?s=400&v=4",
+        "roles": [
+            "Skript maintainer",
+            "Skript Developer"
+        ]
+    },
+    "FranKusmiruk": {
+        "avatar": "https://avatars3.githubusercontent.com/u/48283695?s=400&v=4",
+        "roles": [
+            "Skript Developer"
+        ]
+    },
+    "Pikachu920": {
+        "avatar": "https://avatars0.githubusercontent.com/u/28607612?s=400&v=4",
+        "roles": [
+            "Skript Developer"
+        ]
+    },
+    "Nicofisi": {
+        "avatar": "https://avatars2.githubusercontent.com/u/12614053?s=400&v=4",
+        "roles": [
+            "Skript Developer"
+        ]
+    },
+    "TheBentoBox": {
+        "avatar": "https://avatars3.githubusercontent.com/u/6902229?s=400&v=4",
+        "roles": [
+            "Issue tracker manager",
+            "Aliases developer"
```

Comment:
Consistency seems a little off in terms of casing. 
You have `Skript Developer` and then `Skript maintainer`
I recommend sticking with upper casing for each word, but thats just my opinion. 
If not, then lower case developer to match the rest.